.github/actions/deploy_v2/action.yml

```diff
@@ -0,0 +1,132 @@
+name: deploy_v2
+description: Deploys application to AKS
+
+inputs:
+  arm-access-key:
+    required: true
+  azure-credentials:
+    required: true
+  environment:
+    description: Environment to deploy to
+    required: true
+  pr-number:
+    description: PR number for the review app
+    required: false
+  sha:
+    description: Commit sha to be deployed
+    required: true
+  slack-webhook:
+    required: true
+
+outputs:
+  deploy-url:
+    value: ${{ steps.set_env_var.outputs.deploy_url }}
+
+runs:
+  using: composite
+  steps:
+    - name: Set Environment variables
+      id: set_env_var
+      shell: bash
+      run: |
+        tf_vars_file=terraform/workspace_variables/${{ inputs.environment }}.tfvars.json
+        cluster=$(jq -r '.cluster' ${tf_vars_file})
+
+        if [ -n "${{ inputs.pr-number }}" ]; then
```

Comment:
The logic in this block differs from the equivalent logic in `actions/deploy`.  Do we want production to fallback to a `teacherservices.cloud` domain?  If we do is that a temporary or permanent requirement?

Reply:
Commented on this logic